src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1594,6 +1650,43 @@ private function addHttpClientSection(ArrayNodeDefinition $rootNode)
                                             ->variableNode('md5')->end()
                                         ->end()
                                     ->end()
+                                    ->arrayNode('retry_failed')
```

Comment:
Can we move the copy/pasted code in a method instead? If not, the same changes proposed above should be applied here as well.

Reply:
agreed. 
done